src/devices/Pcd8544/README.md

```diff
@@ -0,0 +1,156 @@
+
+Here is an example on how you can convert an existing image to extract the raw data:
+
+```csharp
+Bitmap bitmapNokia = new(Path.Combine("nokia_bw.bmp"), true);
+var bitmap2 = BitmapToByteArray(bitmapNokia);
+lcd.SetByteMap(bitmap2);
+lcd.Refresh();
+
+byte[] BitmapToByteArray(Bitmap bitmap)
+{
+    if ((bitmap.Width != Pcd8544.PixelScreenSize.Width) || (bitmap.Height != Pcd8544.PixelScreenSize.Height))
+    {
+        throw new ArgumentException($"{nameof(bitmap)} should be same size as the screen {Pcd8544.PixelScreenSize.Width}x{Pcd8544.PixelScreenSize.Height}");
+    }
+
+    byte[] toReturn = new byte[Pcd8544.ScreenSizeBytes];
+    int width = Pcd8544.Size.Width;
+    Color colWhite = Color.FromArgb(255, 255, 255, 255);
+    for (int position = 0; position < Pcd8544.ScreenSizeBytes; position++)
+    {
+        byte toStore = 0;
+        for (int bit = 0; bit < 8; bit++)
+        {
+            toStore = (byte)(toStore | ((bitmap.GetPixel(position % width, position / width * 8 + bit) == colWhite ? 0 : 1) << bit));
+        }
+
+        toReturn[position] = toStore;
+    }
+
+    return toReturn;
+}
+```
+
+In case you want to convert existing images which have a different size than the 84x48 screen size, you have to resize the picture like this:
+
+```csharp
+// Open a non bitmap and resize it
+Bitmap bitmapLarge = new(Image.FromFile(Path.Combine("nonbmp.jpg")), Pcd8544.PixelScreenSize);
+Bitmap newBitmapSmall1color = bitmapLarge.Clone(new Rectangle(0, 0, Pcd8544.PixelScreenSize.Width, Pcd8544.PixelScreenSize.Height), System.Drawing.Imaging.PixelFormat.Format1bppIndexed);
+var bitmap3 = BitmapToByteArray(newBitmapSmall1color);
+```
+
+Note: you may want to reverse the colors first depending on what you want.
+
+## Advance functions
+
+You can adjust couple of factors like `Bias`,  `Temperature`, `Contrast` and `Brightness`. Teh [samples](./samples/Program.cs) will run thru all of them so you can understand the impact of each of them.
+
+In general, it is recommended to leave Temperature to the 0 coefficient, if you are in normal conditions. The Bias can be left to 4 as well as a default value for normal conditions.
+
+Then you can use the Contrast property to properly adjust the contrast. A value around 0x30 is usually a good one for normal conditions.
+
+But you may have to adjust those depending on the conditions of lights, temperature you are in.
+
+### Brightness
+
+The brightness is controlling the PWM. If you have not passed any PWM controller, this has no effect on the screen.
+Brightness goes from 0.0 to 0.1f.
+
+```csharp
+lcd.BacklightBrightness = 0.2f;
+```
+
+### InverseMode
+
+An invert mode is available, it just revert the screen colors. So white pixels become black and vice versa
+
+```csharp
+lcd.InverseMode = true;
+```
+
+### Enabled
+
+You can switch on or off the screen fully with the `Enabled` property:
+
+```csharp
+lcd.Enabled = true;
```

Comment:
I'd suggest renaming this because Enabled doesn't suggest it will turn it off (perhaps just `On`)

Reply:
For all other bindings we have `Enabled` for those scenarios, so just wanted to align. I can clearly rename `On` but that would not align with our general approach for other bindings.